deploy/internal/nsfs-pvc-cr.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: default
```

Comment:
I agree with @achouhan09 on the name. let's set here the name we intend to use later: `noobaa-default-nsfs-pvc`

Reply:
done.